cpp/unreal_plugins/UrdfRobot/Source/UrdfRobot/UrdfRobotComponent.cpp

```diff
@@ -0,0 +1,307 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfRobot/UrdfRobotComponent.h"
+
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <Containers/Array.h>       // TArray
```

Comment:
```
#include <Containers/Array.h>
```

Comment not needed because the type that is introduced is the same as the name of the header.